test/engine/SpatialJoinPrefilterTest.cpp

```diff
@@ -684,6 +685,32 @@ TEST(SpatialJoinTest, BoundingBoxPrefilterDeactivatedTooLargeBox) {
                           7,
                           1},
                          INTERSECTS, true);
+
+  // Update runtime parameter for second test
+  EXPECT_EQ(RuntimeParameters().get<"spatial-join-prefilter-max-size">(),
+            2'500);
```

Comment:
This check seems odd. If this condition is important for the test, just set it explicitly, or skip the test (there's a GTEST macro for that) if it's not that important in this case.
There also exists a helper function `setRuntimeInfoForTesting` or something along those lines that does automatically reset the value to its original when it goes out of scope.